### ISSUE TITLE
Hide password length during key derivation

### ISSUE THREAD

**Issue by fschwebel** [PhilippC/keepass2android#471]
After typing the master password, considering replacing the password characters asterisks by either an empty string or a defined number of asterisks. This is in order to avoid leaking the length of the master password on screen during the period of time the device is applying the KDF (that can be long depending on the DB settings). See the VeraCrypt bootloader for an example of implementation - in their case, they fill the input with a large number of asterisks before starting the KDF.

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
I agree .This is why most Linux shows you not any feedback in form of *** while typing. It makes guesses easier.
